tests/transforms/test_transforms.py

```diff
@@ -82,7 +82,8 @@ def test_augmentation_sequential_gray(batch_gray: dict[str, Tensor]) -> None:
         'bbox_xyxy': torch.tensor([[0.0, 0.0, 2.0, 2.0]], dtype=torch.float),
         'labels': torch.tensor([[0, 1]]),
     }
-    augs = K.AugmentationSequential(K.RandomHorizontalFlip(p=1.0), data_keys=None)
+    with pytest.deprecated_call():
+        augs = AugmentationSequential(K.RandomHorizontalFlip(p=1.0), data_keys=None)
```

Comment:
These tests were designed for our wrapper, no need to test the upstream implementation. Should we test our deprecated wrapper or just remove these tests?

Reply:
I think we can go ahead and remove them. Don't see a reason to maintain them now.